docs/dialect/graphblas_dialect_tutorials/graphblas_lower/graphblas_convert_layout.ipynb

```diff
@@ -0,0 +1,628 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "greenhouse-advocacy",
+   "metadata": {},
+   "source": [
+    "# graphblas.convert_layout\n",
+    "\n",
+    "This example will go over how to use the `--graphblas-lower` pass from `graphblas-opt` to lower the `graphblas.convert_layout` op.\n",
+    "\n",
+    "Let’s first import some necessary modules and generate an instance of our JIT engine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "selected-emphasis",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlir_graphblas\n",
+    "import mlir_graphblas.sparse_utils\n",
+    "import numpy as np\n",
+    "\n",
+    "engine = mlir_graphblas.MlirJitEngine()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "promising-fields",
+   "metadata": {},
+   "source": [
+    "Here are the passes we'll use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "executed-third",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "passes = [\n",
+    "    \"--graphblas-lower\",\n",
+    "    \"--sparsification\",\n",
+    "    \"--sparse-tensor-conversion\",\n",
+    "    \"--linalg-bufferize\",\n",
+    "    \"--func-bufferize\",\n",
+    "    \"--tensor-bufferize\",\n",
+    "    \"--tensor-constant-bufferize\",\n",
+    "    \"--finalizing-bufferize\",\n",
+    "    \"--convert-linalg-to-loops\",\n",
+    "    \"--convert-scf-to-std\",\n",
+    "    \"--convert-std-to-llvm\",\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "opening-firewall",
+   "metadata": {},
+   "source": [
+    "The GraphBLAS dialect's ops operate on and usually return sparse matrices. To analyze the results of the ops, it'd be nice to have some functions that'll convert those sparse matrices into dense matrices. The MLIR code below does that. \n",
+    "\n",
+    "We'll have 2 functions, `csr_densify4x4` and `csc_densify4x4`, which will return dense matrices from 4x4 sparse matrices in CSR and CSC formats, respectively. "
```

Comment:
Since this'll require some big changes, I've moved this to https://github.com/metagraph-dev/mlir-graphblas/issues/89 so that it can be handled in another PR.